packages/datadog_flutter_plugin/pubspec.lock

```diff
@@ -5,170 +5,178 @@ packages:
     dependency: transitive
     description:
       name: _fe_analyzer_shared
-      url: "https://pub.dartlang.org"
+      sha256: "569ddca58d535e601dd1584afa117710abc999d036c0cd2c51777fb257df78e8"
```

Comment:
should this file be in the commit if only docs are supposed to be updated?

Reply:
I'm hoping this will fix a build issue...